README.md

```diff
@@ -109,33 +109,33 @@ JDK should be downloaded manually, and the following environment variables shoul
 
 ### Running in Local mode
 
-In the Developer Command Prompt for Visual Studio where `JAVA_HOME` is set properly, navigate to [SparkCLR](./) directory:
+In the Developer Command Prompt for Visual Studio where `JAVA_HOME` is set properly, navigate to [SparkCLR](../../) directory:
 
 ```  
-Runsamples.cmd  
+RunSamples.cmd  
```

Comment:
Change 's' to uppercase to make it consistent with real file name, although file names are case insensitive on Windows